products/pluto/pluto_build/ae_output_scripts/publish.sh

```diff
@@ -5,7 +5,8 @@ set_error_traps
 python3 -m dcpy.utils.s3 \
     --bucket de-sandbox \
     --folder-path ae_output \
-    --s3-path ae_pilot \
+    --s3-path ae-pilot-project \
     --acl public-read \
     --max-files 20000000 \
-    --contents-only
+    --contents-only \
+    --keep-existing
```

Comment:
I've been occasionally doing this in chunks, and they've been adding in additional files so don't want to overwrite

Reply:
just curious how you run it in chunks?

Reply:
I'll say, generate and push zoom levels 9-16 in one go, and then 17 in another action

Reply:
gotcha, I thought that meant chunks within a single zoom level